qemu/tests/balloon_check.py

```diff
@@ -37,6 +37,11 @@ def __init__(self, test, params, env):
             self.params["balloon_test_setup_ready"] = True
             self.ori_gmem = self.get_memory_status()
             self.current_gmem = self.ori_gmem
+            self.pre_mem = self.get_vm_mem(self.vm)
+            self.pre_gmem = self.ori_gmem
+        # ori_mem/gmem is original memory
+        # pre_mem/gmem is memory value before balloon test
+        # current_mem/gmem is current memory
         self.ori_mem = self.get_vm_mem(self.vm)
```

Comment:
Prefer to move the comments before L38, and move L45 after L38, then you can set `pre_mem = ori_mem` instead of invoking get_vm_mem() again.

Reply:
move.